unyt/_unit_lookup_table.py

```diff
@@ -95,10 +97,11 @@
     "lx": (1.0, dimensions.luminous_flux/dimensions.area, 0.0, r"\rm{lx}"),
 
     # Imperial and other non-metric units
-    "in": (0.0254, dimensions.length, 0.0, r"\rm{in}"),
-    "ft": (0.3048, dimensions.length, 0.0, r"\rm{ft}"),
-    "yd": (0.9144, dimensions.length, 0.0, r"\rm{yd}"),
-    "mile": (1609.344, dimensions.length, 0.0, r"\rm{mile}"),
+    "inch": (m_per_inch, dimensions.length, 0.0, r"\rm{in}"),
+    "ft": (m_per_ft, dimensions.length, 0.0, r"\rm{ft}"),
+    "yd": (m_per_ft*3.0, dimensions.length, 0.0, r"\rm{yd}"),
+    "mile": (m_per_ft*5280.0, dimensions.length, 0.0, r"\rm{mile}"),
```

Comment:
Can you leave this as `1609.344`? Otherwise it introduces round-off error level noise in common conversions like mile to km. `1,609.344` is the exact conversion.

Reply:
Similarly for `ft` and `yd` actually, the constants there were hard-coded in there are all exact conversions.